packages/react/src/components/Table/Table.test.jsx

```diff
@@ -173,7 +102,7 @@ describe('Table', () => {
   const expandedData = [
     {
       rowId: 'row-1',
-      content: <RowExpansionContent rowId="row-1" />,
+      content: <div style={{ padding: '8px' }}>Expnded content</div>,
```

Comment:
```suggestion
      content: <div style={{ padding: '8px' }}>Expanded content</div>,
```

Reply:
Thanks, good catch. I need to work on my spelling.  

Regarding the failed test, I'm looking into it but I think it is related to the "load more" PR that we just merged.

Reply:
Ahh.. no of course. It was your fix to the Aggregation column. In the old screenshot it was in the wrong column, that is how I discovered it in the first place.